src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Constants/LoggingConstants.cs

```diff
@@ -14,12 +14,14 @@ internal class LoggingConstants
         public const string RefreshCanceledError = "A refresh operation was canceled.";
         public const string RefreshFailedDueToKeyVaultError = "A refresh operation failed while resolving a Key Vault reference.";
         public const string PushNotificationUnregisteredEndpoint = "Ignoring the push notification received for the unregistered endpoint";
+        public const string QuerySrvDnsFailedError = "Query Srv type Dns failed.";
```

Comment:
Not sure the exact wording, but I think this should be something like "DNS query of type SRV failed."